README.md

```diff
@@ -14,6 +14,18 @@ The focus of these examples is to demonstrate deployment for models trained with
 | [PyTorch Model](./Quick_Deploy/PyTorch/README.md) | [TensorFlow Model](./Quick_Deploy/TensorFlow/README.md) | [ONNX Model](./Quick_Deploy/ONNX/README.md) | [TensorRT Accelerated Model](https://github.com/NVIDIA/TensorRT/tree/main/quickstart/deploy_to_triton) | [vLLM Model](./Quick_Deploy/vLLM/README.md)
 | --------------- | ------------ | --------------- | --------------- | --------------- |
 
+#### Supported Model Table
```

Comment:
I don't think a model necessarily has to have a tutorial to go into this table. Currently it gives the impression that TRTLLM can only support the Llama2 model. Maybe consider adding some more of their example models in this table: https://github.com/NVIDIA/TensorRT-LLM/tree/ffd5af342a817a2689d38e4af2cc59ded877e339/examples. Over time if there is demand we can create corresponding tutorials?

Reply:
Good point. Let me remove this for now.

Reply:
CC @nnshah1

Reply:
maybe we keep the table but figure out a different name.

Example models? ....